factory/usr/lib/systemd/system/initrd-root-device.target.d/42-kernel.conf

```diff
@@ -1,4 +1,3 @@
 [Unit]
-Requires=run-mnt-kernel.mount
```

Comment:
Here we remove the Requires to make the boot more robust. There seems to be issues in ordering when requiring units that do not exist yet. This unit is created later by `the-tool.service`.

There is another PR that cleans up more the graph and removes this file. That because really `initrd-root-device.target` has no reason to depend on `run-mnt-kernel.mount`.

But I wanted to keep this PR minimal, I just had to remove the Requires to make systemd order start jobs properly.